packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -139,7 +169,7 @@ contract L2OutputOracle is Ownable {
         uint256 _l2BlockNumber,
         bytes32 _l1Blockhash,
         uint256 _l1BlockNumber
-    ) external payable onlyOwner {
+    ) external payable onlySequencer {
```

Comment:
We should have a tool that we can mount the key into an image and run to delete outputs. Not a blocker for this PR but logistically if the same key used to submit batches is used to delete outputs we need to be logistically ready. hh tasks require a non significant amount of memory which isn't ideal but it is a standard way to do so, otherside a bash script that uses cast